src/stream/src/task/stream_manager.rs

```diff
@@ -280,13 +280,7 @@ impl LocalStreamManager {
         let (actor_ids_to_send, actor_ids_to_collect) = {
             let core = self.core.lock();
             let actor_ids_to_send = core.context.lock_barrier_manager().all_senders();
-            let actor_ids_to_collect = core
-                .context
-                .actor_infos
```

Comment:
The actor infos are all actors **we know** from this compute node. It may include actors from other compute nodes.